src/pages/baeyoonah/Comment/Comment.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Comment.scss';
+
+class Comment extends React.Component {
+  constructor(props) {
+    super(props);
+    // this.state = {
+    //   id: '',
+    //   comment: [''],
+    // };
+  }
+
+  render() {
+    return (
+      <div className="CommentList">
```

Comment:
- 컴포넌트와 동일하게 클래스네임 지어주시고, 첫글자를 대문자로 작성하는 컨벤션은 컴포넌트를 네이밍 하기 위한 컨벤션 입니다!
```suggestion
      <div className="comment">
```

Reply:
아 이런 실수를 ... 수정했습니다!